packages/fx-core/src/component/driver/deploy/azure/azureStorageStaticWebsiteConfigDriver.ts

```diff
@@ -65,8 +65,8 @@ export class AzureStorageStaticWebsiteConfigDriver implements StepDriver {
 
     const properties = {
       staticWebsite: {
-        indexDocument: input.indexPage,
-        errorDocument404Path: input.errorPage,
+        indexDocument: input.indexPage ?? "index.html",
```

Comment:
Maybe we can define a constant for this default value

Reply:
Ok, changed to DeployConstant